packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -573,13 +573,17 @@ function resolve(
             if (!didWarnAboutDeprecatedWillMount[componentName]) {
               lowPriorityWarning(
                 false,
-                '%s: componentWillMount() is deprecated and will be ' +
-                  'removed in the next major version. Read about the motivations ' +
-                  'behind this change: ' +
-                  'https://fb.me/react-async-component-lifecycle-hooks' +
-                  '\n\n' +
-                  'As a temporary workaround, you can rename to ' +
-                  'UNSAFE_componentWillMount instead.',
+                'componentWillMount has been renamed to UNSAFE_componentWillMount, ' +
+                  "and the old name won't work in the next major version of React.\n" +
+                  'We suggest doing one of the following:\n' +
+                  '- To rename all deprecated lifecycles to their new names, you can run ' +
```

Comment:
I'm concerned about this being our first "suggestion" in the list since it's _not actually_ our primary recommendation.

Reply:
I think people might read this list as if it's prioritized. Could we move the rename bit to the _end_?

Reply:
Can indeed. Will do.

Reply:
done